logos/README.md

```diff
@@ -0,0 +1,10 @@
+# UGM logos
+Cat-themed logos for the 2023 MDAnalysis UGM, created by Fiona Naughton. Copyright (c) 2023 Fiona Naughton.
+
+ - `mdanalysisUGM_catlogo_textless.png`: image only version
+ - `mdanalysisUGM_catlogo_text`: version with text
+
+## License
+Licensed under a Creative Commons [Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/) License.
```

Comment:
I think to be on the safe side we should license under CC-BY-ND (no derivs) so that it's not possible to take the MDA logo out of the cat-based logo (which would be allowed under CC-BY-SA).